config/application.rb

```diff
@@ -33,6 +33,8 @@ class Application < Rails::Application
       config.autoload_paths << Rails.root.join('lib/middleware')
 
       config.middleware.insert_after Rack::Runtime, 'DailyRateLimit' unless Rails.env.test?
+
+      config.cache_store = :redis_store, 'redis://localhost:6379/0/cache', { expires_in: 90.minutes }
```

Comment:
Line is too long. [101/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.